Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Queries/DriverQuote.hs

```diff
@@ -100,3 +100,9 @@ deleteByDriverId personId =
   Esq.delete $ do
     driverQuotes <- from $ table @DriverQuoteT
     where_ $ driverQuotes ^. DriverQuoteDriverId ==. val (toKey personId)
+
+findDriverQuoteBySearchId' :: Transactionable m => Id DSReq.SearchRequest -> DTypeBuilder m (Maybe DriverQuoteT)
```

Comment:
same here

Reply:
In this case we can consider function with `'` for internal use, because it returns `TType`, not domain type